tsconfig.json

```diff
@@ -36,7 +36,7 @@
   ],
   "include": [
     "src/**/*.ts",
-    "src/**/*.tsx"
+    "src/**/*.tsx", "src/**/*.js"
```

Comment:
when `src/utils/isMobileDetect` is converted typescript... we don't need this line.

Reply:
Done, in this [commit](https://github.com/infographicsgroup/sapera-components/pull/26/commits/141dd6b1b9355444ac2eadefe2ba84d8bacbb6fc).